src/definition/populate.go

```diff
@@ -65,7 +67,10 @@ func IntegrationPopulator(config *IntegrationPopulateConfig) (bool, []error) {
 				if nsGetter := config.Specs[groupLabel].NamespaceGetter; nsGetter != nil {
 					ns := nsGetter(metrics)
 					if !config.Filterer.IsAllowed(ns) {
-						continue
+						if groupLabel != NamespaceGroup {
```

Comment:
Here we are breaking the Genericness of the function, but it was the best way we found to add this label